manifests/0000_08_cluster_operator.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  name: ingress
```

Comment:
Bad copypasta!

Reply:
bootstrap-complete timed out, presumably because it was looking for the "ingress" clusteroperator instead of "dns".

Reply:
yes, should be dns